README.md

```diff
@@ -1,12 +1,77 @@
 <!--
-   - SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+   - SPDX-FileCopyrightText: 2019-2021 Serokell <https://serokell.io>
    -
    - SPDX-License-Identifier: LicenseRef-ReplaceMe
    -->
 
-# metatemplates
+**Please read this file as a raw textual file, it has comments for people
+who create a new repo from this template.**
 
-Common files present in our repositories or their templates.
+This repository is a template that we use at Serokell to create new repositories.
+See [GitHub documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template)
+for more details about this feature.
 
-[Templates directory](/templates) contains various files that you should consider adding to your repository.
-See [page on Notion](https://www.notion.so/serokell/How-to-create-and-maintain-a-repository-9028c5c379364407b8b2019b69d2e64a) about repository maintenance.
+If you have just created a new repo from this template, please start from here.
+Here are some things to keep in mind:
+* We have [a page in Notion](https://www.notion.so/serokell/Create-a-repository-9028c5c379364407b8b2019b69d2e64a) describing how to create a new repository.
+* Make sure to read meta-comments (you won't see them in renderred markdown because they are comments, but you can see them in raw files) and remove them eventually.
```

Comment:
```suggestion
* Make sure to read meta-comments (you won't see them in rendered markdown because they are comments, but you can see them in raw files) and remove them eventually.
```